### ISSUE TITLE
SetArray not working

### ISSUE THREAD

**Issue by nikalexey** [spyzhov/ajson#22]
Hello.
I need replace node in array
For example in JSON {"key": [1, 2, 3]} replace second index to another Node
```
vals := parent.MustArray()
vals[2] = anotherNode
parent.SetArray(vals)
```
and as result of Marshal received empty array {"key": []}

**Comment by spyzhov** [comment]
Thanks, I'll check it.

**Comment by spyzhov** [comment]
Thanks! Fixed in [v0.2.4](https://github.com/spyzhov/ajson/releases/tag/v0.2.4)

**Comment by nikalexey** [comment]
Thank you for fast response. =)
